codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -335,8 +334,8 @@ private static void writeChildNode(FlowState state) {
     /**
      * The outbound flow control state for a single stream.
      */
-    final class FlowState {
-        private final Deque<Frame> pendingWriteQueue;
+    private class FlowState {
```

Comment:
A class within the `DefaultHttp2RemoteFlowController` scope could :)  Not that is it too hard to derive but it may provide clarity into the hierarchy intentions/assumptions and also a hint to JIT.